### ISSUE TITLE
Generate training dataset from flood event AOI

### ISSUE THREAD

**Issue by jcahail** [azavea/noaa-flood-mapping#32]
- Produce STAC catalogs from flood event AOI.
- Produce training and testing sets that properly divide the data.

Note that the validation set is going to be generated from the same flood events list.
Care should be taken that neither training or test sets overlap with the validation set.
